isofit/core/forward.py

```diff
@@ -206,13 +206,19 @@ def calc_rdn(self, x, geom, rho_dir_dir=None, rho_dif_dir=None, Ls=None):
         rho_dif_dir_hi = self.upsample(self.surface.wl, rho_dif_dir)
         Ls_hi = self.upsample(self.surface.wl, Ls)
 
-        return self.RT.calc_rdn(x_RT, rho_dir_dir_hi, rho_dif_dir_hi, Ls_hi, geom)
+        return self.RT.calc_rdn(
+            x_RT=x_RT,
+            rho_dir_dir=rho_dir_dir_hi,
+            rho_dif_dir=rho_dif_dir_hi,
+            Ls=Ls_hi,
+            geom=geom,
+        )
```

Comment:
Introducing named args here because RadiativeTransfer.calc_rdn expects them in a different order than ForwardModel.calc_rdn, which could lead to future bugs.